util/test/perf/perfgraph.js

```diff
@@ -250,6 +232,7 @@ function getNextDivs(afterDiv) {
   var annToggle        = addButtonHelper('annotations');
   var screenshotToggle = addButtonHelper('screenshot');
   var closeGraphToggle = addButtonHelper('close');
+  var resetY           = addButtonHelper('reset Y zoom');
```

Comment:
On the graphs the text looks a little off. Maybe lowercase 'Y' or do "reset y-axis zoom", but that starts to get long